Gems/Atom/RPI/Code/Source/RPI.Public/Pass/RenderPass.cpp

```diff
@@ -367,9 +378,14 @@ namespace AZ
 
         ViewPtr RenderPass::GetView() const
         {
-            if (m_flags.m_hasPipelineViewTag && m_pipeline)
+            if (m_pipeline)
             {
-                const AZStd::vector<ViewPtr>& views = m_pipeline->GetViews(m_viewTag);
+                const auto& viewTag = GetPipelineViewTag();
+                if (viewTag.IsEmpty())
+                {
+                    return nullptr;
+                }
+                const AZStd::vector<ViewPtr>& views = m_pipeline->GetViews(viewTag);
                 if (views.size() > 0)
                 {
                     return views[0];
```

Comment:
This block can be replaced by m_pipeline->GetFirstView(viewTag)

Reply:
done